### ISSUE TITLE
Crash when spruce & jungle bits in same blockspace (v9.18 1.9)

### ISSUE THREAD

**Issue by joni1** [ChiselsAndBits/Chisels-and-Bits#52]
---- Minecraft Crash Report ----

WARNING: coremods are present:
  LoadingPlugin (Quark-beta-18.jar)
  CodeChickenCorePlugin (CodeChickenCore-1.9-2.0.1.53-universal.jar)
  CCCDeobfPlugin (unknown)
  FMLPlugin (InventoryTweaks-1.61-dev-49.jar)
  CCLCorePlugin (CodeChickenLib-1.9-2.0.1.23-universal.jar)
Contact their authors BEFORE contacting forge

// You're mean.

Time: 01.06.16 19:21
Description: Unexpected error

java.lang.IndexOutOfBoundsException: Index: 0, Size: 0
    at java.util.ArrayList.rangeCheck(ArrayList.java:653)
    at java.util.ArrayList.get(ArrayList.java:429)
    at mod.chiselsandbits.helpers.ContinousBits.getItem(ContinousBits.java:72)
    at mod.chiselsandbits.network.packets.PacketChisel.doAction(PacketChisel.java:165)
    at mod.chiselsandbits.core.ClientSide.drawHighlight(ClientSide.java:714)
    at net.minecraftforge.fml.common.eventhandler.ASMEventHandler_148_ClientSide_drawHighlight_DrawBlockHighlightEvent.invoke(.dynamic)
    at net.minecraftforge.fml.common.eventhandler.ASMEventHandler.invoke(ASMEventHandler.java:49)
    at net.minecraftforge.fml.common.eventhandler.EventBus.post(EventBus.java:140)
    at net.minecraftforge.client.ForgeHooksClient.onDrawBlockHighlight(ForgeHooksClient.java:139)
    at net.minecraft.client.renderer.EntityRenderer.func_175068_a(EntityRenderer.java:1329)
    at net.minecraft.client.renderer.EntityRenderer.func_78471_a(EntityRenderer.java:1229)
    at net.minecraft.client.renderer.EntityRenderer.func_181560_a(EntityRenderer.java:1038)
    at net.minecraft.client.Minecraft.func_71411_J(Minecraft.java:1072)
    at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:366)
    at net.minecraft.client.main.Main.main(SourceFile:124)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:483)
    at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
    at net.minecraft.launchwrapper.Launch.main(Launch.java:28)
## A detailed walkthrough of the error, its code path and all known details is as follows:

-- Head --
Stacktrace:
    at java.util.ArrayList.rangeCheck(ArrayList.java:653)
    at java.util.ArrayList.get(ArrayList.java:429)
    at mod.chiselsandbits.helpers.ContinousBits.getItem(ContinousBits.java:72)
    at mod.chiselsandbits.network.packets.PacketChisel.doAction(PacketChisel.java:165)
    at mod.chiselsandbits.core.ClientSide.drawHighlight(ClientSide.java:714)
    at net.minecraftforge.fml.common.eventhandler.ASMEventHandler_148_ClientSide_drawHighlight_DrawBlockHighlightEvent.invoke(.dynamic)
    at net.minecraftforge.fml.common.eventhandler.ASMEventHandler.invoke(ASMEventHandler.java:49)
    at net.minecraftforge.fml.common.eventhandler.EventBus.post(EventBus.java:140)
    at net.minecraftforge.client.ForgeHooksClient.onDrawBlockHighlight(ForgeHooksClient.java:139)
    at net.minecraft.client.renderer.EntityRenderer.func_175068_a(EntityRenderer.java:1329)
    at net.minecraft.client.renderer.EntityRenderer.func_78471_a(EntityRenderer.java:1229)

-- Affected level --
Details:
    Level name: MpServer
    All players: 1 total; [EntityPlayerSP['Darth_Eggedosis'/1297486, l='MpServer', x=295,97, y=18,00, z=405,16]]
    Chunk stats: MultiplayerChunkCache: 961, 961
    Level seed: 0
    Level generator: ID 00 - default, ver 1. Features enabled: false
    Level generator options: 
    Level spawn location: World: (115,64,188), Chunk: (at 3,4,12 in 7,11; contains blocks 112,0,176 to 127,255,191), Region: (0,0; contains chunks 0,0 to 31,31, blocks 0,0,0 to 511,255,511)
    Level time: 27064450 game time, 5685889 day time
    Level dimension: 0
    Level storage version: 0x00000 - Unknown?
    Level weather: Rain time: 0 (now: false), thunder time: 0 (now: false)
    Level game mode: Game mode: creative (ID 1). Hardcore: false. Cheats: false
    Forced entities: 135 total; [EntityPlayerSP['Darth_Eggedosis'/1297486, l='MpServer', x=295,97, y=18,00, z=405,16], EntityChicken['Chicken'/1297683, l='MpServer', x=370,89, y=17,00, z=421,79], EntityChicken['Chicken'/1297680, l='MpServer', x=374,05, y=17,00, z=421,41], EntityChicken['Chicken'/1297686, l='MpServer', x=371,39, y=17,00, z=415,83], EntityChicken['Chicken'/1297684, l='MpServer', x=375,06, y=17,00, z=418,42], EntityChicken['Chicken'/1297691, l='MpServer', x=376,48, y=17,00, z=416,59], EntityChicken['Chicken'/1297689, l='MpServer', x=376,54, y=17,00, z=423,11], EntityChicken['Chicken'/1297688, l='MpServer', x=371,02, y=17,00, z=417,53], EntityChicken['Chicken'/1297694, l='MpServer', x=372,50, y=17,00, z=416,34], EntityChicken['Chicken'/1297693, l='MpServer', x=370,89, y=17,00, z=422,76], EntityChicken['Chicken'/1297692, l='MpServer', x=371,97, y=17,00, z=415,83], EntityZombie['Zombie'/1297667, l='MpServer', x=238,75, y=19,00, z=472,52], EntityCreeper['Creeper'/1297666, l='MpServer', x=247,50, y=24,00, z=474,50], EntityChicken['Chicken'/1297664, l='MpServer', x=376,31, y=17,00, z=416,16], EntityChicken['Chicken'/1297671, l='MpServer', x=372,50, y=17,00, z=421,71], EntityChicken['Chicken'/1297669, l='MpServer', x=369,37, y=17,00, z=394,79], EntityBat['Bat'/1297668, l='MpServer', x=261,43, y=12,15, z=477,22], EntityChicken['Chicken'/1297674, l='MpServer', x=371,32, y=17,00, z=416,65], EntityArmorStand['Armor Stand'/282, l='MpServer', x=319,50, y=17,00, z=426,50], EntityChicken['Chicken'/1297673, l='MpServer', x=372,73, y=17,00, z=423,11], EntityDummy['entity.testdummy.Dummy.name'/283, l='MpServer', x=292,50, y=17,00, z=419,50], EntityChicken['Chicken'/1297672, l='MpServer', x=371,79, y=17,00, z=423,11], EntityChicken['Chicken'/1297679, l='MpServer', x=372,24, y=17,00, z=421,06], EntityChicken['Chicken'/1297678, l='MpServer', x=371,32, y=17,00, z=423,11], EntityChicken['Chicken'/1297677, l='MpServer', x=372,29, y=17,00, z=422,73], EntityArmorStand['Armor Stand'/287, l='MpServer', x=285,50, y=17,00, z=426,50], EntityChicken['Chicken'/1297676, l='MpServer', x=371,60, y=17,00, z=419,54], EntityArmorStand['Armor Stand'/288, l='MpServer', x=287,50, y=17,00, z=426,50], EntityChicken['Chicken'/1297715, l='MpServer', x=371,92, y=17,00, z=420,56], EntityChicken['Chicken'/1297714, l='MpServer', x=373,01, y=17,00, z=422,01], EntityChicken['Chicken'/1297718, l='MpServer', x=371,93, y=17,00, z=421,57], EntityChicken['Chicken'/1297717, l='MpServer', x=371,80, y=17,00, z=415,34], EntityChicken['Chicken'/1297722, l='MpServer', x=375,10, y=17,00, z=416,32], EntityChicken['Chicken'/1297720, l='MpServer', x=370,89, y=17,00, z=421,22], EntityChicken['Chicken'/1297699, l='MpServer', x=376,08, y=17,00, z=417,16], EntityChicken['Chicken'/1297698, l='MpServer', x=374,12, y=17,00, z=422,78], EntityChicken['Chicken'/1297697, l='MpServer', x=371,05, y=17,00, z=418,10], EntityChicken['Chicken'/1297696, l='MpServer', x=370,89, y=17,00, z=417,07], EntitySquid['Squid'/1297197, l='MpServer', x=286,04, y=45,35, z=415,78], EntityItem['item.item.dyePowder.black'/1297491, l='MpServer', x=324,83, y=33,00, z=413,22], EntityChicken['Chicken'/1297495, l='MpServer', x=263,31, y=14,71, z=428,69], EntityBoat['Boat'/1297494, l='MpServer', x=263,31, y=14,52, z=428,69], EntitySquid['Squid'/1297492, l='MpServer', x=335,49, y=59,98, z=407,55], EntityArmorStand['Armor Stand'/1297498, l='MpServer', x=256,50, y=17,00, z=426,50], EntityItem['item.item.feather'/1297497, l='MpServer', x=263,34, y=15,09, z=429,27], EntityCreeper['Creeper'/1297501, l='MpServer', x=279,94, y=15,00, z=436,42], EntityChicken['Chicken'/1297728, l='MpServer', x=376,49, y=17,00, z=422,13], EntitySkeleton['Skeleton'/1297733, l='MpServer', x=354,50, y=19,00, z=470,50], EntityItemFrame['entity.ItemFrame.name'/1297531, l='MpServer', x=191,97, y=56,50, z=395,50], EntityItemFrame['entity.ItemFrame.name'/1297532, l='MpServer', x=182,50, y=55,50, z=400,03], EntitySquid['Squid'/1297505, l='MpServer', x=294,72, y=59,06, z=390,24], EntityCreeper['Creeper'/1297775, l='MpServer', x=224,50, y=29,00, z=481,50], EntityBat['Bat'/1297774, l='MpServer', x=375,96, y=14,31, z=341,66], EntityCreeper['Creeper'/1286508, l='MpServer', x=274,48, y=11,00, z=419,18], EntitySquid['Squid'/1297516, l='MpServer', x=324,24, y=53,74, z=420,07], EntitySkeleton['Skeleton'/1297772, l='MpServer', x=362,50, y=14,00, z=333,50], EntityEnderman['Enderman'/1297555, l='MpServer', x=232,47, y=15,00, z=332,71], EntityBoat['Boat'/1297559, l='MpServer', x=228,11, y=62,52, z=364,42], EntityBoat['Boat'/1297558, l='MpServer', x=224,84, y=62,52, z=365,25], EntityZombie['Zombie'/1297557, l='MpServer', x=226,59, y=27,00, z=350,21], EntitySkeleton['Skeleton'/1297556, l='MpServer', x=235,50, y=20,00, z=343,23], EntityArmorStand['Armor Stand'/1297563, l='MpServer', x=238,50, y=17,88, z=399,50], EntityBoat['Boat'/1297562, l='MpServer', x=235,70, y=62,52, z=373,58], EntityBoat['Boat'/1297561, l='MpServer', x=229,69, y=62,52, z=373,43], EntityBoat['Boat'/1297560, l='MpServer', x=234,57, y=62,52, z=366,30], EntityArmorStand['Armor Stand'/1297567, l='MpServer', x=245,50, y=17,00, z=396,50], EntitySkeleton['Skeleton'/1297566, l='MpServer', x=244,49, y=5,00, z=397,32], EntityArmorStand['Armor Stand'/1297565, l='MpServer', x=238,50, y=17,88, z=402,50], EntityArmorStand['Armor Stand'/1297564, l='MpServer', x=238,50, y=17,88, z=405,50], EntitySlime['Slime'/1297546, l='MpServer', x=327,30, y=17,00, z=427,25], EntityArmorStand['Armor Stand'/1297545, l='MpServer', x=321,50, y=17,00, z=426,50], EntityZombie['Zombie'/1297544, l='MpServer', x=316,50, y=28,00, z=446,50], EntityCreeper['Creeper'/1297550, l='MpServer', x=216,84, y=74,00, z=382,49], EntitySquid['Squid'/1297587, l='MpServer', x=290,24, y=53,57, z=368,14], EntityArmorStand['Armor Stand'/1297591, l='MpServer', x=245,50, y=17,00, z=404,50], EntityArmorStand['Armor Stand'/1297590, l='MpServer', x=245,50, y=17,00, z=402,50], EntityArmorStand['Armor Stand'/1297589, l='MpServer', x=245,50, y=17,00, z=400,50], EntityItem['item.item.dyePowder.black'/1297588, l='MpServer', x=312,20, y=38,00, z=376,38], EntityCreeper['Creeper'/1297595, l='MpServer', x=296,81, y=22,00, z=456,47], EntityEnderman['Enderman'/1297594, l='MpServer', x=304,27, y=21,00, z=451,45], EntityArmorStand['Armor Stand'/1297593, l='MpServer', x=254,50, y=17,00, z=426,50], EntityZombie['Zombie'/1297592, l='MpServer', x=322,24, y=28,00, z=442,51], EntityCreeper['Creeper'/1297599, l='MpServer', x=250,50, y=14,00, z=444,50], EntityBat['Bat'/1297598, l='MpServer', x=325,73, y=12,04, z=362,39], EntityBat['Bat'/1297597, l='MpServer', x=299,52, y=24,18, z=465,90], EntityCreeper['Creeper'/1297596, l='MpServer', x=316,49, y=29,00, z=455,79], EntityGuardian['Guardian'/1297571, l='MpServer', x=254,39, y=43,00, z=385,18], EntityCreeper['Creeper'/1297570, l='MpServer', x=255,50, y=21,00, z=385,50], EntityArmorStand['Armor Stand'/1297569, l='MpServer', x=245,50, y=17,00, z=394,50], EntityArmorStand['Armor Stand'/1297568, l='MpServer', x=245,50, y=17,00, z=398,50], EntityItem['item.item.dyePowder.black'/1297575, l='MpServer', x=302,84, y=33,00, z=433,33], EntitySkeleton['Skeleton'/1297574, l='MpServer', x=299,49, y=24,00, z=442,79], EntityZombie['Zombie'/1297573, l='MpServer', x=294,50, y=22,00, z=444,50], EntityBoat['Boat'/1297572, l='MpServer', x=291,25, y=14,52, z=432,31], EntityBat['Bat'/1297576, l='MpServer', x=267,75, y=9,10, z=407,75], EntityCreeper['Creeper'/1297619, l='MpServer', x=309,50, y=26,00, z=479,50], EntityCreeper['Creeper'/1297618, l='MpServer', x=304,50, y=22,00, z=465,50], EntityGuardian['Guardian'/1297617, l='MpServer', x=287,32, y=39,00, z=466,03], EntityZombie['Zombie'/1297620, l='MpServer', x=308,50, y=23,00, z=469,50], EntitySkeleton['Skeleton'/1297603, l='MpServer', x=344,50, y=17,00, z=447,70], EntitySkeleton['Skeleton'/1297602, l='MpServer', x=339,50, y=15,00, z=440,50], EntitySkeleton['Skeleton'/1297601, l='MpServer', x=344,50, y=17,00, z=447,04], EntityBoat['Boat'/1297600, l='MpServer', x=350,66, y=14,52, z=432,31], EntityWitch['Witch'/1297607, l='MpServer', x=359,50, y=30,00, z=401,50], EntityItem['item.item.egg'/1297606, l='MpServer', x=359,14, y=17,00, z=402,56], EntitySkeleton['Skeleton'/1297605, l='MpServer', x=328,50, y=27,00, z=448,73], EntityZombie['Zombie'/1297604, l='MpServer', x=343,25, y=26,00, z=438,53], EntityChicken['Chicken'/1297611, l='MpServer', x=365,16, y=17,00, z=397,75], EntitySquid['Squid'/1297610, l='MpServer', x=336,27, y=56,25, z=366,84], EntityBat['Bat'/1297609, l='MpServer', x=343,90, y=24,84, z=349,61], EntityCreeper['Creeper'/1297608, l='MpServer', x=352,51, y=32,00, z=407,17], EntityBat['Bat'/1281999, l='MpServer', x=276,36, y=15,10, z=421,53], EntityBat['Bat'/1297615, l='MpServer', x=362,90, y=30,10, z=425,33], EntityChicken['Chicken'/1297614, l='MpServer', x=359,49, y=14,55, z=431,52], EntityBoat['Boat'/1297613, l='MpServer', x=352,31, y=14,52, z=428,69], EntityBat['Bat'/1297612, l='MpServer', x=300,79, y=20,96, z=459,11], EntityChicken['Chicken'/1297649, l='MpServer', x=375,48, y=17,00, z=414,89], EntityChicken['Chicken'/1297648, l='MpServer', x=373,26, y=17,00, z=414,89], EntityChicken['Chicken'/1297655, l='MpServer', x=372,13, y=17,00, z=415,87], EntityChicken['Chicken'/1297658, l='MpServer', x=371,33, y=17,00, z=415,31], EntityChicken['Chicken'/1297656, l='MpServer', x=374,52, y=17,00, z=415,93], EntityChicken['Chicken'/1297663, l='MpServer', x=371,76, y=17,00, z=414,89], EntityGuardian['Guardian'/1294819, l='MpServer', x=316,67, y=33,00, z=397,09], EntitySkeleton['Skeleton'/1297635, l='MpServer', x=343,78, y=25,00, z=450,50], EntityChicken['Chicken'/1297639, l='MpServer', x=372,76, y=17,00, z=414,89], EntityCreeper['Creeper'/1297638, l='MpServer', x=364,50, y=20,00, z=359,50], EntityBat['Bat'/1293029, l='MpServer', x=317,25, y=15,10, z=394,50], EntityZombie['Zombie'/1297637, l='MpServer', x=361,50, y=26,00, z=440,50], EntityCreeper['Creeper'/1297636, l='MpServer', x=342,50, y=25,00, z=448,50], EntityChicken['Chicken'/1297643, l='MpServer', x=371,21, y=17,00, z=415,73], EntityChicken['Chicken'/1297642, l='MpServer', x=372,75, y=17,00, z=415,33], EntityChicken['Chicken'/1297640, l='MpServer', x=371,28, y=17,00, z=414,89], EntityChicken['Chicken'/1297646, l='MpServer', x=370,89, y=17,00, z=416,32], EntityChicken['Chicken'/1297645, l='MpServer', x=372,24, y=17,00, z=414,89], EntityChicken['Chicken'/1297644, l='MpServer', x=370,89, y=17,00, z=415,31]]
    Retry entities: 0 total; []
    Server brand: fml,forge
    Server type: Non-integrated multiplayer server
Stacktrace:
    at net.minecraft.client.multiplayer.WorldClient.func_72914_a(WorldClient.java:409)
    at net.minecraft.client.Minecraft.func_71396_d(Minecraft.java:2647)
    at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:395)
    at net.minecraft.client.main.Main.main(SourceFile:124)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:483)
    at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
    at net.minecraft.launchwrapper.Launch.main(Launch.java:28)

-- System Details --
Details:
    Minecraft Version: 1.9
    Operating System: Windows 7 (amd64) version 6.1
    Java Version: 1.8.0_25, Oracle Corporation
    Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
    Memory: 629386744 bytes (600 MB) / 1449598976 bytes (1382 MB) up to 4281597952 bytes (4083 MB)
    JVM Flags: 6 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xmx4G -XX:+UseConcMarkSweepGC -XX:+CMSIncrementalMode -XX:-UseAdaptiveSizePolicy -Xmn128M
    IntCache: cache: 0, tcache: 0, allocated: 13, tallocated: 95
    FML: MCP 9.23 Powered by Forge 12.16.1.1907 24 mods loaded, 24 mods active
    States: 'U' = Unloaded 'L' = Loaded 'C' = Constructed 'H' = Pre-initialized 'I' = Initialized 'J' = Post-initialized 'A' = Available 'D' = Disabled 'E' = Errored
    UCHIJA  mcp{9.19} [Minecraft Coder Pack](minecraft.jar) 
    UCHIJA  FML{8.0.99.99} [Forge Mod Loader](forge-1.9-12.16.1.1907.jar) 
    UCHIJA  Forge{12.16.1.1907} [Minecraft Forge](forge-1.9-12.16.1.1907.jar) 
    UCHIJA  CodeChickenCore{2.0.1.53} [CodeChicken Core](minecraft.jar) 
    UCHIJA  additionalbanners{1.1.0.11} [Additional Banners](AdditionalBanners-1.9-1.1.0.13.jar) 
    UCHIJA  bagginses{2.3.0b} [Bagginses](Bagginses-1.9-2.3.0b.jar) 
    UCHIJA  BiomesOPlenty{4.0.2.2018} [Biomes O' Plenty](BiomesOPlenty-1.9-4.0.2.2018-universal.jar) 
    UCHIJA  JEI{3.3.3.200} [Just Enough Items](jei_1.9-3.3.3.200.jar) 
    UCHIJA  BloodMagic{1.9-2.0.0-38} [Blood Magic: Alchemical Wizardry](BloodMagic-1.9-2.0.0-38.jar) 
    UCHIJA  Chameleon{1.9-2.1.3} [Chameleon](Chameleon-1.9-2.1.3.jar) 
    UCHIJA  ChickenChunks{2.0.1.26} [ChickenChunks](ChickenChunks-1.9-2.0.1.26-universal.jar) 
    UCHIJA  chiselsandbits{9.18} [Chisels & Bits](chiselsandbits-9.18.jar) 
    UCHIJA  craftingcraft{3.0.3} [CraftingCraft](craftingcraft-mc1.9-3.0.3.jar) 
    UCHIJA  shetiphiancore{3.2.2} [ShetiPhian-Core](shetiphiancore-1.9.0-3.2.2.jar) 
    UCHIJA  endertanks{1.3.3} [EnderTanks](endertanks-1.9.0-1.3.3.jar) 
    UCHIJA  inventorytweaks{1.61-dev-49-c5b6395} [Inventory Tweaks](InventoryTweaks-1.61-dev-49.jar) 
    UCHIJA  mantle{1.9-0.10.1.jenkins141} [Mantle](Mantle-1.9-0.10.1.jar) 
    UCHIJA  testdummy{1.10} [MmmMmmMmmMmm](MmmMmmMmmMmm-1.10.jar) 
    UCHIJA  harvestcraft{1.9a} [Pam's HarvestCraft](Pam's HarvestCraft 1.9b.jar) 
    UCHIJA  Quark{beta-18} [Quark](Quark-beta-18.jar) 
    UCHIJA  StorageDrawers{1.9-3.1.6} [Storage Drawers](StorageDrawers-1.9-3.1.6.jar) 
    UCHIJA  tconstruct{1.9-2.3.1.jenkins225} [Tinkers' Construct](TConstruct-1.9-2.3.1.jar) 
    UCHIJA  Waila{1.7.0} [Waila](Waila-1.7.0-B2_1.9.jar) 
    UCHIJA  worldedit{6.1.2} [WorldEdit](worldedit-forge-mc1.9-6.1.2-dist.jar) 
    Loaded coremods (and transformers): 
LoadingPlugin (Quark-beta-18.jar)
  vazkii.quark.base.asm.ClassTransformer
CodeChickenCorePlugin (CodeChickenCore-1.9-2.0.1.53-universal.jar)
  codechicken.core.asm.TweakTransformer
  codechicken.core.asm.DelegatedTransformer
  codechicken.core.asm.DefaultImplementationTransformer
CCCDeobfPlugin (unknown)

FMLPlugin (InventoryTweaks-1.61-dev-49.jar)
  invtweaks.forge.asm.ContainerTransformer
CCLCorePlugin (CodeChickenLib-1.9-2.0.1.23-universal.jar)
  codechicken.lib.asm.ClassHeirachyManager
  codechicken.lib.asm.RenderHookTransformer
    GL info: ' Vendor: 'ATI Technologies Inc.' Version: '4.5.13416 Compatibility Profile Context 15.300.1025.0' Renderer: 'AMD Radeon HD 6900 Series'
    Pulsar/tconstruct loaded Pulses: 
        - TinkerCommons (Enabled/Forced)
        - TinkerWorld (Enabled/Not Forced)
        - TinkerTools (Enabled/Not Forced)
        - TinkerSmeltery (Enabled/Not Forced)
        - TinkerGadgets (Enabled/Not Forced)
        - TinkerOredict (Enabled/Forced)
        - TinkerIntegration (Enabled/Forced)
        - TinkerFluids (Enabled/Forced)
        - TinkerMaterials (Enabled/Forced)
        - chiselsandbitsIntegration (Enabled/Not Forced)

```
Launched Version: 1.9-forge1.9-12.16.1.1907
LWJGL: 2.9.4
OpenGL: AMD Radeon HD 6900 Series GL version 4.5.13416 Compatibility Profile Context 15.300.1025.0, ATI Technologies Inc.
GL Caps: Using GL 1.3 multitexturing.
```

Using GL 1.3 texture combiners.
Using framebuffer objects because OpenGL 3.0 is supported and separate blending is supported.
Shaders are available because OpenGL 2.1 is supported.
VBOs are available because OpenGL 1.5 is supported.

```
Using VBOs: No
Is Modded: Definitely; Client brand changed to 'fml,forge'
Type: Client (map_client.txt)
Resource Packs: 
Current Language: English (US)
Profiler Position: N/A (disabled)
CPU: 4x Intel(R) Core(TM) i5-4690K CPU @ 3.50GHz
```

Here is my crash and this is for the version 9.18 of the mod for minecraft version 1.9. I hope this get's fixes soon! Good Luck!

Edit: Note this seems for me to only be the case when placing bits in draw region mode when there are bits from other blocks in the same blockspace.

**Comment by AlgorithmX2** [comment]
Any other clarification possible?

I tried using draw region to put different bits in the same block on both 1.9 and 1.9.4 so far I've not been able to reproduce the issue.

**Comment by joni1** [comment]
Try using modded bits, I provoked my crash using dark oak bark bit drawn alongside spruce wood bit

**Comment by AlgorithmX2** [comment]
So far I've had no luck reproducing the issue, please verify that you can reproduce it either by it self, or with one mod. ( and let me know which mod you tested it with if that is the case )

Also is there any thing else you can give me? Anything in your off hand? does it work when your inventory is empty ( minus the bit ), are you using a mod that alters your inventory in some fashion?

I must be missing some critical piece of information that makes this crash possible.

I've tried testing with BOP, and Quark so far. ( BOP for a modded block to test with )

**Comment by joni1** [comment]
Try having a diamond chisel in your main hand and any bit in your off hand and using draw region, I can reproduce this crash any day of the week... I've made this crash with bits found from standard chisel and bits (from vanilla blocks) using the offhand, but I've made this happen with quarks dark oak bark bit in the main hand and nothing in the offhand. Note that draw region won't work in a blockspace with no other bits in it (Note that this is having dia chisel in main hand and stone bit in off hand)...

**Comment by AlgorithmX2** [comment]
Thank you for the clarification, I was finally able to reproduce the crash. I should have a fix out today.

**Comment by AlgorithmX2** [comment]
Fixed e3258dfd315ca2beb284640ee9874db4ea9cc7d4

**Comment by joni1** [comment]
Cheers!
